src/providers/colors/ColorProvider.hpp

```diff
@@ -13,6 +13,10 @@ enum class ColorType {
     Whisper,
     RedeemedHighlight,
     FirstMessageHighlight,
+    AnnouncePurple,
+    AnnounceBlue,
+    AnnounceGreen,
+    AnnounceOrange
```

Comment:
Notice that before you added extra members to this enum, the very last member already had a trailing comma. Doing this in enum/list/string literals is allowed and considered a good practice here which usually makes formatting + potential future diffs cleaner.
```suggestion
    AnnounceOrange,
```